core/process/last_status_spec.rb

```diff
@@ -0,0 +1,19 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+
+ruby_version_is '2.5' do
+  describe 'Process#last_status' do
+    it 'returns the status of the last executed child process in the current thread' do
+      pid = Process.spawn("ruby", "-e", "exit 13")
```

Comment:
This should use the current ruby (with `[*ruby_exe, '-e', 'exit 13']`) as there might not be a ruby in $PATH.
Alternatively, just `Process.spawn("exit 0")` should be portable and simpler (from #454).

Reply:
done. I have updated the PR to use `Process.spawn("exit 0")`